### ISSUE TITLE
Fix lmh installation for groups

### ISSUE THREAD

**Issue by rappatoni** [UniFormal/MMT#494]
Only archives (corresponding to git repositories) should have this option.

EDIT: It appears that groups should be installable. This does not appear to work at the moment, the repositories within a group can only be installed individually.

**Comment by Jazzpirate** [comment]
you'll need to be more specific, because it works flawlessly for me. I've called
`lmh install smglom` several times over the last weeks, last time today.

**Comment by rappatoni** [comment]
@Jazzpirate: indeed, smglom works for me too. Can you confirm that this is a Windows-only issue by attempting to clone e.g. the group Test?

**Comment by rappatoni** [comment]
@Jazzpirate: the issue appears to only apply to the Mathhub side pane. "lmh install" through the mmt-shell works flawlessly for groups, through the side pane it produces the above error for some groups.

E.g.:

lmh: Attempting to install archive FrameIT
lmh: attempting to 'git clone https://gl.mathhub.info/FrameIT.git' into 'C:\Users\Max\Uni\MMT-Archives\FrameIT'
lmh: trying to clone FrameIT
error: (lmh) git clone https://gl.mathhub.info/FrameIT.git failed
error: (lmh) installation has failed, please make sure that git is installed and try again. 
user: 'lmh install FrameIT'
  lmh: fetching https://gl.mathhub.info/api/v4/projects?per_page=100&page=1
  lmh: fetching https://gl.mathhub.info/api/v4/projects?per_page=100&page=2
  lmh: fetching https://gl.mathhub.info/api/v4/projects?per_page=100&page=3
  lmh: fetching https://gl.mathhub.info/api/v4/projects?per_page=100&page=4
  lmh: Attempting to install archive FrameIT/FrameIT
  lmh: attempting to 'git clone https://gl.mathhub.info/FrameIT/FrameIT.git' into 'C:\Users\Max\Uni\MMT-Archives\FrameIT\FrameIT'
  lmh: trying to clone FrameIT/FrameIT
  lmh: git clone successful
    user: opened archive C:\Users\Max\Uni\MMT-Archives\FrameIT\FrameIT
  lmh: checking for dependencies of FrameIT/FrameIT
    lmh: installing dependency /meta-inf of FrameIT/FrameIT
    lmh: Attempting to install archive /meta-inf
    lmh: attempting to 'git clone https://gl.mathhub.info//meta-inf.git' into 'C:\Users\Max\Uni\MMT-Archives\meta-inf'
    lmh: trying to clone /meta-inf
    error: (lmh) git clone https://gl.mathhub.info//meta-inf.git failed
    error: (lmh) installation has failed, please make sure that git is installed and try again. 
user: 'lmh install FrameIT' finished


It is noteworthy that the shell command "lmh install" triggers a https-clone here whereas the side pane install attempts an ssh-clone. So maybe the problem is that the side pane clicky thing passes on an ssh-URL and sometimes a https-URL is needed?
